### ISSUE TITLE
The date is shown nowhere

### ISSUE THREAD

**Issue by Brutus5000** [visit1985/mdp#144]
Hi there,

I just used mdp for the first time. I was wondering what the purpose of the %date variable is. Neither in the sample file nor in my own files the date is shown anywhere on the screen. I would have expected in the bottom line as the names are also there, maybe centered?

Is this a bug or is this intentional?

**Comment by visit1985** [comment]
Yes, this is a bug. It is supposed to show up when `--noslidenum` is used.
But the name of the variable does not really matter, Just the fact, that it is the third one, should show it in the bottom right corner.
